server/test/test.js

```diff
@@ -31,15 +31,15 @@ describe('Save and load tests', () => {
     const testText = 'Testing rest-api'
```

Comment:
Get /Userlistille pitäis myös kirjottaa testi.

Reply:
Tehty.